frontend/app/routes/page-components/requests/submission-detail-form.tsx

```diff
@@ -289,11 +289,11 @@ export function SubmissionDetailsForm({
             )}
             <InputField
               className="w-full"
-              id="alternate-contact-email-address"
-              name="alternateContactEmailAddress"
+              id="additional-contact-email-address"
+              name="additionalContact.businessEmailAddress"
```

Comment:
we can keep it simple without using dot in the name, but that would need to be changed in the actions as well...
```suggestion
              name="additionalContactBusinessEmailAddress"
```